google/s2geometry
# Giant update

* Add new files
      * s2buffer_operation
      * s2builderutil_get_snapped_winding_delta
      * s2builderutil_lax_polyline_layer
      * s2memory_tracker
      * s2shapeutil_conversion
      * s2winding_operation
      * s2wrapped_shape
      * thread_testing
    * Expose some new functions in pywraps2
    * Remove internal absl; depend on released absl
    * Temporarily require C++17 (until someone complains)
    * Require SWIG 4
    * Remove internal files that are no longer needed
    * Improved documentation of EncodedS2ShapeIndex
